aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -1771,6 +1771,11 @@ Tensor _sparse_sum_backward_cpu(const Tensor& grad_, const SparseTensor& input_,
   TORCH_CHECK(!grad_.is_cuda(), "_sparse_sum_backward_cpu: expected 'grad_' to be CPU tensor, but got CUDA tensor");
   TORCH_CHECK(!input_.is_cuda(), "_sparse_sum_backward_cpu: expected 'input_' to be CPU tensor, but got CUDA tensor");
 
+  // Short circuit if grad is either zero or empty.
+  if ((grad_.layout() != kStrided && !grad_._nnz()) || !grad_.numel()) {
```

Comment:
Nit: it would be safer to use:
```suggestion
  if (((grad_.is_sparse() || at::sparse_csr::is_sparse_compressed(grad_)) && !grad_._nnz()) || !grad_.numel()) {
```